<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>GuiTeleportRoom</class>
 <widget class="QDialog" name="GuiTeleportRoom" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="mp_gbTeleport" >
     <property name="title" >
      <string>Teleport</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="2" column="0" >
       <widget class="QLabel" name="mp_labelFlags" >
        <property name="toolTip" >
         <string>Locazione di destinazione del teleport</string>
        </property>
        <property name="text" >
         <string>Flags</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="mp_comboSector" >
        <property name="minimumSize" >
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="mp_labelToRoom" >
        <property name="minimumSize" >
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Locazione di destinazione del teleport</string>
        </property>
        <property name="text" >
         <string>Alla stanza</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3" >
       <widget class="QLineEdit" name="mp_leFlags" >
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="mp_labelSector" >
        <property name="minimumSize" >
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>Settore</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="mp_leTeleportTime" >
        <property name="maxLength" >
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="mp_labelTeleportCount" >
        <property name="minimumSize" >
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Fa scattare la teleport quando qualcuno resta nella stanza teleport_count volte il teleport_time.</string>
        </property>
        <property name="text" >
         <string>Contatore</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="mp_labelTeleportTime" >
        <property name="toolTip" >
         <string>La teleport scatta ogni teleport_time.</string>
        </property>
        <property name="text" >
         <string>Tempo</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="4" >
       <widget class="QToolButton" name="mp_tbFlags" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QLineEdit" name="mp_leToRoom" >
        <property name="minimumSize" >
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="maxLength" >
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="4" >
       <widget class="QToolButton" name="mp_tbRoom" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QLineEdit" name="mp_leTeleportCount" >
        <property name="maxLength" >
         <number>4</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mp_pbOk" >
       <property name="text" >
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mp_pbSave" >
       <property name="text" >
        <string>Applica</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mp_pbCancel" >
       <property name="text" >
        <string>Annulla</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>mp_leTeleportTime</tabstop>
  <tabstop>mp_leTeleportCount</tabstop>
  <tabstop>mp_comboSector</tabstop>
  <tabstop>mp_leToRoom</tabstop>
  <tabstop>mp_tbRoom</tabstop>
  <tabstop>mp_tbFlags</tabstop>
  <tabstop>mp_pbOk</tabstop>
  <tabstop>mp_pbSave</tabstop>
  <tabstop>mp_pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
